docs/_includes/generated/checkpointing_configuration.html

```diff
@@ -60,7 +54,19 @@
             <td><h5>state.savepoints.dir</h5></td>
             <td style="word-wrap: break-word;">(none)</td>
             <td>String</td>
-            <td>The default directory for savepoints. Used by the state backends that write savepoints to file systems (MemoryStateBackend, FsStateBackend, RocksDBStateBackend).</td>
+            <td>The default directory for savepoints. Used by the state backends that write savepoints to file systems (HashMapStateBackend, EmbeddedRocksDBStateBackend).</td>
+        </tr>
+        <tr>
+            <td><h5>state.snapshot.fs.memory-threshold</h5></td>
```

Comment:
How about `state.storage.fs.memory-threshold` instead of  `state.snapshot.fs.memory-threshold`?

ditto `write-buffer-size`